MapleServer2/Packets/Helpers/ItemPacketHelper.cs

```diff
@@ -36,7 +36,7 @@ public static class ItemPacketHelper {
                 pWriter.WriteTemplate();
             }
 
-            if (item.InventoryType == InventoryTab.Pets) {
+            if (item.InventoryTab == InventoryType.Pets) {
```

Comment:
This whole `Tab` vs `Type` is kind of confusing, maybe we should just pick one and stick with it throughout?